server/src/database/schema.prisma

```diff
@@ -488,25 +490,30 @@ model PipelineProgress {
   closeDate   DateTime?
   probability Int?
 
-  pipeline         Pipeline                 @relation(fields: [pipelineId], references: [id])
+  pipeline         Pipeline      @relation(fields: [pipelineId], references: [id])
   pipelineId       String
-  pipelineStage    PipelineStage            @relation(fields: [pipelineStageId], references: [id])
+  pipelineStage    PipelineStage @relation(fields: [pipelineStageId], references: [id])
   pipelineStageId  String
-  pointOfContact   Person?                  @relation(fields: [pointOfContactId], references: [id])
+  pointOfContact   Person?       @relation("PointOfContactPipelineProgress", fields: [pointOfContactId], references: [id])
   pointOfContactId String?
-  progressableType PipelineProgressableType
-  progressableId   String
+
   /// @TypeGraphQL.omit(input: true, output: true)
-  workspace        Workspace                @relation(fields: [workspaceId], references: [id])
+  workspace   Workspace @relation(fields: [workspaceId], references: [id])
   /// @TypeGraphQL.omit(input: true, output: true)
-  workspaceId      String
+  workspaceId String
 
   /// @TypeGraphQL.omit(input: true, output: true)
   deletedAt DateTime?
 
   createdAt DateTime @default(now())
   updatedAt DateTime @updatedAt
 
+  companyId String?
+  company   Company? @relation(fields: [companyId], references: [id])
+
+  personId String?
+  person   Person? @relation(fields: [personId], references: [id])
```

Comment:
We may want to enforce having one and only one of `[companyId, personId]` set at this level.